libs/common/byteutils.hpp

```diff
@@ -18,31 +18,43 @@
 #ifndef IROHA_BYTEUTILS_H
 #define IROHA_BYTEUTILS_H
 
-#include <crypto/base64.hpp>
+#include <string>
+
+#include <nonstd/optional.hpp>
 
+#include "crypto/base64.hpp"
 extern "C" {
-#include <crypto/lookup3.h>
+#include "crypto/lookup3.h"
 }
-
-#include <string>
-#include "types.hpp"
+#include "common/types.hpp"
 
 namespace iroha {
 
   /**
-   * Converts given string to the blob of given size.
-   * @tparam size
-   * @param s
-   * @return
+   * Create blob_t from string of specified size
+   * @tparam size - expected size of string
```

Comment:
Also add that size is the size of blob_t, and it is expected that the size of string is the same